crates/turbopack-ecmascript/src/parse/mod.rs

```diff
@@ -330,6 +334,10 @@ async fn parse_content(
                 is_typescript,
             ));
 
+            let mut tla_visitor = TopLevelAwaitVisitor::default();
+
+            parsed_program.visit_with(&mut tla_visitor);
```

Comment:
Wasn't sure when to check for it, so I've done it before it gets transformed in case preset env gets rid of it 🤔

Reply:
It must be after transforming. Transforms might add new TLAs, so we want to see them.

I would put it into `AnalyzeEcmascriptModuleResult` instead of `ParseResult`